### ISSUE TITLE
Can't send Item price to analytics

### ISSUE THREAD

**Issue by artemkhalygov** [invertase/react-native-firebase#5227]
## Issue

Why it's impossible to add `price` to the `Item` inside `viewItemList`? As I can see in documentation https://firebase.google.com/docs/analytics/measure-ecommerce#javascript - it should be possible. 

![image](https://user-images.githubusercontent.com/10270025/116413113-ff9fc980-a83f-11eb-815d-193f9f2df252.png)
---

But the current `Item` structure doesn't have a price.  Maybe I'm wrong but it should be here, isn't it?

``` 
export interface Item {
    item_brand?: string;
    item_id?: string;
    item_name?: string;
    item_category?: string;
    item_category2?: string;
    item_category3?: string;
    item_category4?: string;
    item_category5?: string;
    item_list_id?: string;
    item_list_name?: string;
    item_location_id?: string;
    item_variant?: string;
    quantity?: number;
 }
```

Thanks.

### Javascript

<details><summary>Click To Expand</summary>
<p>

#### `package.json`:

```json
    "@react-native-firebase/analytics": "^11.2.0",
    "@react-native-firebase/app": "^11.2.0",
    "@react-native-firebase/dynamic-links": "^11.2.0",
    "@react-native-firebase/messaging": "^11.2.0",
```

**Comment by mikehardy** [comment]
Have you checked the issues list? There is some good discussion on this one if I recall correctly

**Comment by artemkhalygov** [comment]
@mikehardy  Yes, I am. But I found only this PR that related to my issue https://github.com/invertase/react-native-firebase/pull/4536

I'm almost sure that price also should be added.

**Comment by mikehardy** [comment]
I will happily work with you to merge a PR implementing the things you need, the PR you linked could be a good template to do so

You may be interested in the comments on a related issue, starting here: https://github.com/invertase/react-native-firebase/issues/4578#issuecomment-738299069

**Comment by artemkhalygov** [comment]
@mikehardy Agree with you, that I can fix it myself

So I created a PR https://github.com/invertase/react-native-firebase/pull/5232

**Comment by mikehardy** [comment]
Fixed by #5232
